pyteal/ast/cond.py

```diff
@@ -9,16 +10,34 @@
     from pyteal.compiler import CompileOptions
 
 
+def reformat_multi_argv(argv: tuple[list[Expr], ...]) -> list[list[Expr]]:
```

Comment:
@jdtzmn Minor:  Do you intend for `reformat_multi_argv` to be used by library users?

My sense is _no_.  If the method is intended for internal usage, can you rename to `_reformat_multi_argv`?

The leading `_` signifies internal usage as discussed in https://peps.python.org/pep-0008/#descriptive-naming-styles (see `_single_leading_underscore`).

I'm nitpicking because once exposed publicly, we cannot easily make changes.

Reply:
To expand on this topic with my personal views:
* I view it as _essential_ to use a leading underscore for a non-public function name that is exported by the `pyteal` package. Commonly this would be static or instance methods on classes which are not designed for public use.
* I don't think it's as important for non-public functions which are not exported by the `pyteal` package (and therefore users can't access them) to use a leading underscore. I would never consider this wrong, as the underscore does help indicate the private nature of such a function to us developers, but I don't view it as necessarily required.

Reply:
@jdtzmn Thanks for bringing in the changes - resolves the thread from my perspective.

@jasonpaulos I agree with the rationale.  Let's continue discussion out-of-band to the PR.